packages/integration/lib/src/views/activity/notification_screen.dart

```diff
@@ -0,0 +1,283 @@
+                child: widget.emptyNotificationFeedViewBuilder?.call(context) ??
+                    _defEmptyNotificationFeedView(),
+              ),
+              itemBuilder: (context, item, index) {
+                return widget.notificationTileBuilder
+                        ?.call(context, item, _defNotificationTile(item)) ??
+                    _defNotificationTile(item);
+              },
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _defEmptyNotificationFeedView() {
+    LMFeedThemeData feedThemeData = LMFeedCore.theme;
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          LMFeedIcon(
+            type: LMFeedIconType.icon,
+            icon: Icons.post_add,
+            style: LMFeedIconStyle(
+              size: 48,
+              color: feedThemeData.onContainer,            ),
+          ),
+          const SizedBox(height: 12),
+          LMFeedText(
+            text: 'No notifications to show',
+            style: LMFeedTextStyle(
+              textStyle: TextStyle(
+                fontSize: 20,
+                fontWeight: FontWeight.bold,
+                color: feedThemeData.onContainer,
+              ),
+            ),
+          ),
+         SizedBox(
+          height: 48,
+         )
+        ],
+      ),
+    );
+  }
+
+  LMFeedNotificationTile _defNotificationTile(
+      LMNotificationFeedItemViewData item) {
+    return LMFeedNotificationTile(
+      notificationItemViewData: item,
+      style: LMFeedNotificationTileStyle.basic().copyWith(
+        activeBackgroundColor: _theme.disabledColor.withOpacity(0.5),
+        crossAxisAlignment: CrossAxisAlignment.start,
+        padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 20),
+      ),
+      onTap: () {
+        if (!item.isRead) {
+          _notificationsBloc?.add(
+            LMFeedMarkNotificationAsReadEvent(
+              activityId: item.id,
+            ),
+          );
+        }
+        debugPrint(item.cta ?? 'no cta');
+        routeNotification(item.cta ?? '');
+      },
+    );
+  }
+
+  Widget _defCustomWidget() {
+    return Padding(
+      padding: const EdgeInsets.only(left: 20, right: 20, top: 10, bottom: 20),
+      child: LMFeedText(
+          text: 'Notifications',
+          style: LMFeedTextStyle(
+              textStyle: TextStyle(
+            fontFamily: 'Inter',
+            fontSize: 20,
+            fontWeight: FontWeight.w600,
+          ))),
+    );
+  }
+
+  Widget _defNotificationsErrorView(String message) {
+    return Padding(
+      padding: const EdgeInsets.all(20),
+      child: LMFeedText(
+        text: message,
+        style: LMFeedTextStyle(
+          textAlign: TextAlign.center,
+          textStyle: TextStyle(
+            fontSize: 18,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+    );
+  }
+
+  void routeNotification(String route) async {
```

Comment:
Don't we have this function built in `LMNotificationHandler` ?

Reply:
No, this function is related to notification feed and not push notifications